samples/notebooks/Lightweight Python components - basics.ipynb

```diff
@@ -278,7 +278,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.4"
+   "version": "3.7.0"
```

Comment:
This file changes does not seem to be related to the PR.

Reply:
Left over from a previous PR.
Removed it.